tox.ini

```diff
@@ -21,9 +20,10 @@ deps =
     django19: Django >= 1.9, < 1.10
     django110: Django >= 1.10, < 1.11
     django111: Django >= 1.11, < 1.12
-    django-dev: https://github.com/django/django/tarball/master
+    djangodev: https://github.com/django/django/tarball/master
 commands=
-    python runtests.py
+    coverage run --source polymorphic runtests.py
+    coverage xml -i
```

Comment:
I haven't seen similar results, so I'm not sure why that is, sorry.

Yeah, it's probably better to put this in travis' `after_script`; it's not part of the test itself (and users running locally won't need this).